changelogs/changelog.yaml

```diff
@@ -373,3 +373,15 @@ releases:
         - (#211) Removes codecov from dev dependencies
         - (#203) Adds plugin module
         - (#209) Catches HTTPError for GraphQL query and enables Ansible retries
+
+  4.5.0:
+    changes:
+      bugfixes:
+        - (#223) Inventory Hosts Empty On Error
+        - (#209) Catches HTTPError for `query_graphql` and fails to enable ansible retries
+        - (#228) Fixes graphql inventory grouping by tags
+      minor_changes:
+        - (#211) Removes codecov from dev dependencies
+        - (#203) Adds Plugin Module
```

Comment:
```suggestion
```
I just realized this was listed as part of the changes in 4.4.0 above

Reply:
Should we be removing 203 on this suggestion?

Reply:
Yes, PR 203 is from 4.4.0.